.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 inherit_from:
-- https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop.yml
+- https://raw.githubusercontent.com/ManageIQ/manageiq/master/.rubocop_base.yml
```

Comment:
But then I have to maintain a style across all repos?
I thought having one in a central place was the point